### PR TITLE
Documentation for translate-params functionality

### PR DESCRIPTION
**Pull request by IShotTheSheriff**

### CODE REVIEW

**Comment by IShotTheSheriff** [comment]
@rubenv Hey although I could push directly to master with this one I would really love to get your approval first. As you know in software craftsmanship writing documentation is the hardest part :). Since English is not my native language and website is highly visible I would rather not pollute repo with unchecked documentation.

File: contents/dev-guide/translate-params/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+</span>
+```
+
+* You can make certain expressions more verbose for translator by making aliases for long name variables and expressions.
+
+```html
+<span translate
+      translate-params-date="ctrl.modificationDate | date : 'yyyy-MM-dd'"
+      translate-params-modifier="ctrl.personObject.fullName">
+    Last modified: {{modificationDate}} by {{modifier}}.
```
Comment (rubenv) [inline contents/dev-guide/translate-params/index.md]: This should probably be `{{date}}`?

**Comment by rubenv** [comment]
@IShotTheSheriff Fully understand!

Only found one small error (I think?), see inline comment above.

Also had it proof-read by my co-worker, he's **very** excited, great feature.

**Comment by IShotTheSheriff** [comment]
Yup you were right, variable name fixed.

**Comment by rubenv** [comment]
And it's available online: https://angular-gettext.rocketeer.be/dev-guide/translate-params/

Thanks, *a lot*!

**Comment by IShotTheSheriff** [comment]
Great thanks as well! 

I've checked online and somehow after merge translate params chapter was duplicated. I've pushed fix directly to master this time.

**Comment by rubenv** [comment]
> I've checked online and somehow after merge translate params chapter was duplicated. I've pushed fix directly to master this time.

Computers, they never get it right :-). Good catch!

Blogged about this as well: https://rocketeer.be/blog/2016/06/translate-params/

**Comment by IShotTheSheriff** [comment]
Awesome! I'm really happy that you like it 👍

**Pull request merged**